Apps/AdminWebClient/src/ClientApp/src/services/restEmailAdminService.ts

```diff
@@ -0,0 +1,62 @@
+import { injectable } from "inversify";
+import { IHttpDelegate, IEmailAdminService } from "@/services/interfaces";
+import { Dictionary } from "vue-router/types/router";
+import { ResultType } from "@/constants/resulttype";
+import RequestResult from "@/models/requestResult";
+import Email from '@/models/email';
+
+@injectable()
+export class RestEmailAdminService implements IEmailAdminService {
+  private readonly BETA_REQUEST_BASE_URI: string = "v1/api/EmailAdmin";
```

Comment:
Not sure if this should be EmailAdmin. This looks like the resource is "The email of the Admin". An alternative would be v1/Admin/api/Email. Thoughts?

Reply:
just following on stephen's namings, I thought just Email would make more sense but idk.